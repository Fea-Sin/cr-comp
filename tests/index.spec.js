import React from 'react'
import { mount } from 'enzyme'
import Comp from '../src'


describe('cr-comp', () => {
  it('base test comp', () => {
    const wrapper = mount(
      <Comp />
    )
    expect(wrapper.find('.hello').text()).toBe('COMP IS DONE')
  })
})