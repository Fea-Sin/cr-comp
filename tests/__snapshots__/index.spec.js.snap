// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cr-comp snapshot to match 1`] = `
<div
  class="hello"
>
  COMP IS DONE
</div>
`;
